standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -2632,9 +2629,6 @@ service ThriftHiveMetastore extends fb303.FacebookService
                        throws (1: MetaException o1)
   list<string> get_all_tables(1: string db_name) throws (1: MetaException o1)
 
-  Table get_table(1:string dbname, 2:string tbl_name)
```

Comment:
The `get_table` and `get_table_objects_by_name` is marked as deprecated in HIVE-15062.